services/quiz.js

```diff
@@ -20,6 +21,21 @@ const quizService = {
       author,
       items
     })
+  },
+
+  updateQuiz: async (id, currentUserId, updateData) => {
+    const quiz = await Quiz.findById(id).exec()
+

```

Comment:
I guess you need to handle an error when the quiz wasn't found.

Reply:
It's included in isEntityValid